packages/app-desktop/gui/NoteEditor/utils/resourceHandling.ts

```
@@ -188,6 +205,23 @@ export async function processPastedHtml(html: string, htmlToMd: HtmlToMarkdownHa
+	// Word encodes newlines in alt attributes as HTML entities (&#10; &#13; &#xA; etc.).
+	// These get decoded to literal newline characters by JSDOM when Turndown processes
+	// the HTML. With preserveImageTagsWithSize=true, Turndown returns node.outerHTML
+	// verbatim — embedding literal newlines inside an HTML attribute value, which
+	// breaks the Markdown raw HTML block (a blank line ends the block, making the
+	// parser treat the <img> as plain text). Normalize them to spaces here.
+	html = html.replace(/(\balt=)(["'])([\s\S]*?)\2/gi, (_m, prefix, quote, altText) => {
+		// Replace HTML-encoded newlines/control chars and literal ones with a space
+		// eslint-disable-next-line no-control-regex
+		const normalized = altText
+			.replace(/&#(?:1[0-3]|[0-9]);|&#x[0-9a-fA-F]{1,2};/g, ' ')
+			.replace(/[\r\n\x00-\x08\x0B\x0C\x0E-\x1F\x7F]/g, ' ')
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)